descartes_light/include/descartes_light/impl/gantry_kinematics.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2016, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef DESCARTES_LIGHT_GANTRY_KINEMATICS_H
+#define DESCARTES_LIGHT_GANTRY_KINEMATICS_H
+#include <descartes_light/interface/kinematics_interface.h>
+#include <descartes_light/utils.h>
+#include <Eigen/Dense>
+#include <vector>
+
+namespace descartes_light
+{
+/**
+ * @brief This takes a kinematics object that represents the robot and adds gantry sampling
```

Comment:
It's probably worth mentioning here that this class is for a robots mounted on two-axis gantries, and the gantry axes are assumed to be orthogonal and along the X and Y axes

Reply:
Updated documentation